system-variables.md

```diff
@@ -5227,11 +5227,10 @@ SHOW WARNINGS;
 - Type: Enumeration
 - Default value: `""`
 - Possible values: `""`, `TABLE`, `GLOBAL`
-- By default, Regions are split for a new table when it is being created in TiDB. This variable controls the table Region scattering strategy, determining how TiDB scatters Regions after splitting. This applies to the scenario where data need to be written in batches right after the tables are created in batches, because the newly split Regions can be scattered in TiKV directly and do not have to wait to be scheduled by PD. To ensure the continuous stability of writing data in batches, the `CREATE TABLE` statement returns success only after the Regions are successfully scattered. This makes the statement's execution time multiple times longer than that when you disable this variable. Possible value descriptions are as follows:
-    - `""`: the default value; indicates that TiDB does not scatter table Regions.
+- When the SHARD_ROW_ID_BITS and PRE_SPLIT_REGIONS parameters are set during table creation, the system automatically splits the table into a specified number of Regions upon successful creation. For these Regions, this variable is used to control the scattering strategy after their splitting. TiDB processes the Regions based on the selected scattering strategy.It is important to note that since the table creation operation waits for the scattering process to complete before returning a success status, enabling this variable may significantly increase the execution time of the CREATE TABLE statement. Compared to the scenario where this variable is disabled, the execution time could be several times longer. Possible value descriptions are as follows:
```

Comment:
```suggestion
- When the SHARD_ROW_ID_BITS and PRE_SPLIT_REGIONS parameters are set during table creation, the system automatically splits the table into a specified number of Regions upon successful creation. For these Regions, this variable is used to control the scattering strategy after their splitting. TiDB processes the Regions based on the selected scattering strategy. It is important to note that since the table creation operation waits for the scattering process to complete before returning a success status, enabling this variable may significantly increase the execution time of the CREATE TABLE statement. Compared to the scenario where this variable is disabled, the execution time could be several times longer. Possible value descriptions are as follows:
```